yt_dlp/extractor/nytimes.py

```diff
@@ -125,138 +91,427 @@ class NYTimesIE(NYTimesBaseIE):
+            'description': 'md5:a31fe3b98a8ce7b98aae097730c269cd',
+        },
+        'playlist_count': 8,
     }]
 
+    _TOKEN = 'MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAuNIzKBOFB77aT/jN/FQ+/QVKWq5V1ka1AYmCR9hstz1pGNPH5ajOU9gAqta0T89iPnhjwla+3oec/Z3kGjxbpv6miQXufHFq3u2RC6HyU458cLat5kVPSOQCe3VVB5NRpOlRuwKHqn0txfxnwSSj8mqzstR997d3gKB//RO9zE16y3PoWlDQXkASngNJEWvL19iob/xwAkfEWCjyRILWFY0JYX3AvLMSbq7wsqOCE5srJpo7rRU32zsByhsp1D5W9OYqqwDmflsgCEQy2vqTsJjrJohuNg+urMXNNZ7Y3naMoqttsGDrWVxtPBafKMI8pM2ReNZBbGQsQXRzQNo7+QIDAQAB'
+    _DNS_UUID = '36dd619a-56dc-595b-9e09-37f4152c7b5d'  # uuid -v5 ns:DNS scoop.nyt.net
+    _GRAPHQL_QUERY = '''query VideoQuery($id: String!) {
+  video(id: $id) {
+    ... on Video {
+      bylines {
+        renderedRepresentation
+      }
+      duration
+      promotionalHeadline
+      promotionalMedia {
+        ... on Image {
+          crops {
+            name
+            renditions {
+              name
+              width
+              height
+              url
+            }
+          }
+        }
+      }
+      renditions {
+        type
+        width
+        height
+        url
+        bitrate
+      }
+      summary
+    }
+  }
+}'''
+
+    def _build_playlist(self, media_items):
+        entries = []
+        for media_id in media_items:
+            json_obj = traverse_obj(self._json_from_graphql(media_id) or {}, ('data', 'video'))
+
+            title = json_obj.get('promotionalHeadline') or media_id
+            description = json_obj.get('summary')
+            creators = ', '.join(
+                [remove_start(creator, 'By ') for creator in traverse_obj(
+                    json_obj, ('bylines', ..., 'renderedRepresentation'))])
+            duration = int_or_none(json_obj.get('duration'))
+            media_content = json_obj.get('renditions')
+            formats, subtitles = self._extract_media_from_json(media_id, media_content)
+
+            thumbnails = []
+            for image in traverse_obj(json_obj, ('promotionalMedia', 'crops', ..., 'renditions', ...)):
+                image_url = image.get('url')
+                if not url_or_none(image_url):
+                    continue
+                thumbnails.append({
+                    'name': image.get('name'),
+                    'url': image_url,
+                    'width': int_or_none(image.get('width')),
+                    'height': int_or_none(image.get('height')),
+                })
+
+            entries.append({
+                'id': media_id,
+                'title': title,
+                'description': description,
+                'duration': duration,
+                'creator': creators,
+                'formats': formats,
+                'subtitles': subtitles,
+                'thumbnails': thumbnails,
+            })
+        return entries
+
+    def _json_from_graphql(self, media_id):
+        # reference: `id-to-uri.js`
+        namespace = uuid.UUID(self._DNS_UUID)
```

Comment:
the class var could be `_DNS_NAMESPACE = uuid.UUID('36dd619a-56dc-595b-9e09-37f4152c7b5d')`